### PR TITLE
 STYLE: replace unwanted-patterns-strings-to-concatenate with ruff #5…

### PR DESCRIPTION
**Pull request by harsimran44**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -199,6 +199,8 @@ update-check = false
 target-version = "py38"
 
 select = [
+  # implicit string concatenation
+  "ISC"
```
Comment (MarcoGorelli) [inline pyproject.toml:203]: needs a comma at the end of the line - did you try running this?

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
thanks for working on this

File: pyproject.toml (hunk 1 above)
Comment (harsimran44) [reply pyproject.toml]: Hello when i am running the **pre-commit run ruff --all-files** . its shows this output. Can u help me with running please.
Capture

Comment (MarcoGorelli) [reply pyproject.toml]: please check https://pandas.pydata.org/docs/dev/development/contributing_environment.html

**Pull request closed without merging**